src/Presentation/Nop.Web/Areas/Admin/Controllers/CustomerController.cs

```diff
@@ -568,7 +571,10 @@ public virtual async Task<IActionResult> Edit(CustomerModel model, bool continue
                 {
                     customer.AdminComment = model.AdminComment;
                     customer.IsTaxExempt = model.IsTaxExempt;
-
+                    
+                    //welcome message
```

Comment:
remove unnecessary comment, when i read line 576 for me is clear that this is welcome message

Reply:
and check all other places